src/Interpreter.test.js

```diff
@@ -141,6 +142,58 @@ test('Do a command with a program', (done) => {
 });
 
 test('Doing an unknown command rejects with Error', () => {
-    const interpreter = new Interpreter();
+    const interpreter = new Interpreter(()=>{});
     return expect(interpreter.doCommand('unknown-command')).rejects.toThrow('Unknown command: unknown-command');
 });
+
+test('Use the callback function on continueRun', () => {
```

Comment:
Please make 3 variants of this test:

- An empty program: "onRunningStateChange is called on run() empty program"
- A program with one step: "onRunningStateChange is called on run() program with one step"
- A program with two steps "onRunningStateChange is called on run() program with two steps"